pubsub/subscriber/subscriber.go

```diff
@@ -181,9 +185,9 @@ func (s *subscriber) Stop(ctx context.Context) error {
 		}
 	}
 
-	s.logger.Logf(log.InfoLevel, "All tasks are finished. Disconnecting from transport.")
+	s.logger.Logf(log.InfoLevel, "All tasks are finished.")
 
-	return s.transport.Disconnect(ctx)
```

Comment:
removal of `transport.Disconnect` here makes Stop() method useless because it won't be able to complete ever. Previously called at the end `transport.Disconnect` of `Stop()` was closing consume channel and thus breaking the main loop above.  Try to call `Stop()` and you will see.

Reply:
it's outdated either since Stop method is dropped in this PR